src/main/java/Lumiere.java

```diff
@@ -0,0 +1,163 @@
+import java.util.*;
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+public class Lumiere {
+    private static String fileLocation = "./lumiere.txt";
+    private static Path filePath = Paths.get(fileLocation);
+
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+
+        List<Task> list = loadTasksFromFile();
```

Comment:
Maybe a more descriptive name could be used here? 
```suggestion
        List<Task> tasks = loadTasksFromFile();
```